multicompany_property_base/models/res_partner.py

```diff
@@ -32,31 +32,45 @@ def _compute_properties(self):
                 values.append(val.id)
             record.property_ids = values
 
+    @api.model
+    def _get_property_value_sanitized(self, p_values):
+        """
+        :return: dict containing the sanitized prorperty_ids value received in the write
+        method
+        The value received should be list, each value being a list or tuple of 3
+        elements, just as in the meth:write format
+        """
+        result = dict()
+        property_obj = self.env["res.partner.property"]
+        for p_value in p_values:
+            is_list = isinstance(p_value, list)
+            is_tuple = isinstance(p_value, tuple)
+            if (not is_list and not is_tuple) or (
+                (is_list or is_tuple) and len(p_value) != 3
```

Comment:
```suggestion
                len(p_value) != 3
```

if the code gets to this point is because it is already a list or a tuple for sure

Reply:
yes, you are right 👍🏽